@navikt/core/css/form/combobox.css

```
@@ -243,6 +246,10 @@
   background-color: var(--ac-combobox-list-item-selected-bg, var(--a-surface-selected));
 }
 
+.navds-combobox__list-item--selected p {
```

Tror den bare trengs å settes på wrapper (for lavere specificity)
```suggestion
.navds-combobox__list-item--selected {
```

---

Nei, `.navds-body-short` overstyrer den, dessverre. Var litt usikker på å bruke `p` eller om jeg burde bruke klassenavnet, men vi må i hvert fall ha en høyere spesifisitet enn `.navds-body-short`. (Arv fra parent har lavere spesifisitet)